test/powershell/Language/Parser/LineContinuance.Tests.ps1

```diff
@@ -0,0 +1,247 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+Describe 'Line Continuance' -Tags 'CI' {
+    BeforeAll {
+        function ExecuteCommand {
+            param ([string]$command)
+            [powershell]::Create().AddScript($command).Invoke()
+        }
+    }
+
+    Context 'Lines ending with a backtick that parse and execute without error' {
+        It 'Lines ending with a single backtick' {
+            $script = @'
+'Hello' + `
+    ' world'
+'@
+            ExecuteCommand $script | Should -Be 'Hello world'
+        }
+
+        It 'Lines ending with a single backtick followed by whitespace' {
+            $script = @'
+# The next line ends with trailing whitespace
+'Hello' + `    
+    ' world'
+'@
+            ExecuteCommand $script | Should -Be 'Hello world'
+        }
+
+        It 'Lines ending with a single backtick followed by whitespace and a comment' {
+            $script = @'
+'Hello' + ` # You can place comments after whitespace following backticks
+    ' world'
+'@
+            ExecuteCommand $script | Should -Be 'Hello world'
+        }
+
+        It 'Lines ending with a single backtick followed by a comment line and then the continued line' {
+            $script = @'
+'Hello' + `
+# You can place comments in the middle of a continued line
+    ' world'
+'@
+            ExecuteCommand $script | Should -Be 'Hello world'
+        }
+    }
+
+    Context 'Lines ending with a backtick that do not parse' {
+        It 'Lines ending with a single backtick followed immediately by a comment' {
+            $script = @'
+'Hello' + `# This is not a valid comment
```

Comment:
I'd prefer it works.

Reply:
This isn't a change from this PR though, is it?Backtick before a comment-start token escapes it. :)

Reply:
Correct. I was looking for line continuance tests, and there weren't any, so I put these in this PR so that we have something to start with.

Also, backtick is an escape character, so in this case I think the behavior is absolutely correct, since the backtick escapes the #. Forcing users to add a space here isn't a big deal. Better that we make it so that backticks hardly ever need to be used, other than intentionally escaping a character, and I think we're almost there (missing automatic line continuance for parameters on newlines, so that will come in another PR, but with an experimental flag since it would technically be a breaking change for commands that start with "-").

Reply:
An example that _would_ parse would be: ```Write-Host Hello `# World```